content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -325,7 +328,8 @@ kubeadm relies on the kubelet to pull the control plane images and run them prop
 After the control plane is up, kubeadm completes the tasks described in following paragraphs.
 
 ### (optional) Write base kubelet configuration
-{{< feature-state for_k8s_version="v1.9" state="alpha" >}}
+
+{{< feature-state for_k8s_version="v1.11" state="beta" >}}
```

Comment:
the 1.9 == alpha state here is about the DynamicKubeletConfig feature gate, not the kubelet v1beta1 API.

Reply:
According to https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/, `DynamicKubeletConfig` has become beta in 1.11.

Reply:
right, i got confused.